packages/core/src/lib/markdown-it/plugins/markdown-it-icons.ts

```diff
@@ -1,65 +1,97 @@
-import octicons, { IconName } from '@primer/octicons';
-import cheerio from 'cheerio';
-
+const octicons = require('@primer/octicons');
+const cheerio = require('cheerio');
```

Comment:
any reason why the import syntax is changed here?

Reply:
ESLint does not allow mix of CommonJS module syntax (module.exports = 'foo' or exports.bar = 'hi') with ECMAScript 6 (ES6) new syntax for modules, using import and export keywords 
```
import foo from './foo';
export function bar() { /* ... */ };
```